tests/test_generate_diff.py

```diff
@@ -5,8 +5,8 @@
 
 def test_generate_diff_json() -> None:
```

Comment:
when you put old files back, makes sense to keep this test and add `plain` to it (don't forget to add tests for nested cases)